src/sonic-config-engine/minigraph.py

```diff
@@ -808,7 +809,11 @@ def parse_meta(meta, hname):
                     cloudtype = value
                 elif name == "ResourceType":
                     resource_type = value
-    return syslog_servers, dhcp_servers, ntp_servers, tacacs_servers, mgmt_routes, erspan_dst, deployment_id, region, cloudtype, resource_type
+                elif name == "KubernetesEnabled":
+                    kube_data["disable"] = value
```

Comment:
>disable [](start = 31, length = 7)

I am confused, do you mean "enabled" ?

Reply:
Name changed to be more appropriate. Good catch. No logical code change still.